app/services/dos_to_unix_lines.rb

```diff
@@ -0,0 +1,19 @@
+class DosToUnixLines
+  def initialize source_filepath, destination_filepath, chunk_size = 2
```

Comment:
The default `chunk_size` is too small. Let's have a better default value. Something like 1MB. You can use `1.megabyte`